llvm/include/llvm/Support/AlwaysTrue.h

```diff
@@ -0,0 +1,29 @@
+//===--- AlwaysTrue.h - Helper for oqaque truthy values        --*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the getNonFoldableAlwaysTrue helper funtion
```

Comment:
```suggestion
// This file defines the getNonFoldableAlwaysTrue helper function.
```
But really, this comment seems redundant.

Reply:
Deleted